### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

**Review by MarkZuber — APPROVED** [review]

**Review by henrik-me — APPROVED** [review]
:shipit:

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
+Version 4.4.3
+===============
+- **When using ADAL v4.4.2 and MSAL v2.6 in the same Xamarin project, a Duplicate Java type error was encountered**. The Android AuthenticationActivity defined name has been changed and is now distinct from the one in MSAL. [ADAL issue](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1479)
+- **When using brokered authentication, if the UID was not present, broker was unable to find the UID in the Android package**. Now ADAL sends the UID and package name as part of the broker options so silent authentication with broker works consistently. [ADAL PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1468)
```
Comment (jmprieur) [inline changelog.txt:4]: What is UID? the user id? upn?

**Review by jmprieur — APPROVED** [review]
LGTM. Thanks @jennyf19

File: changelog.txt (hunk 1 above)
Comment (jennyf19) [reply changelog.txt:4]: @jmprieur https://developer.android.com/reference/android/os/Process#myUid%28%29

**Pull request merged**